### PR TITLE
update contributing documentation

### PR DESCRIPTION
**Pull request by xalopp**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Setup
+
+We recommend following setup:
```
Comment (mmoll) [inline CONTRIBUTING.md]: I think a tad more nativespeaky would be "We **do** recommend **the** following setup"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* make sure that [Composer](https://getcomposer.org) is installed
+* clone the [PHP_CodeSniffer repository](https://github.com/squizlabs/PHP_CodeSniffer)
+* run `composer install` inside the `PHP_CodeSniffer` directory to install required dependencies
+* symlink, copy or clone the [Symfony Coding Standard](https://github.com/djoos/Symfony-coding-standard) and symlink, copy or move its `Symfony` subdirectory to `PHP_CodeSniffer/src/Standards/Symfony`
```
Comment (mmoll) [inline CONTRIBUTING.md]: I know this was copypasted from the README, but the first "symlink, copy or clone" could just get replaced by "get" now.

File: README.md [hunk 3]
```diff
@@ -52,18 +52,7 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
-
-        bin/phpcs --ignore='*/vendor/*'
-
-The unit-tests are run from within the PHP_CodeSniffer directory
-
-* clone the [CodeSniffer repository](https://github.com/squizlabs/PHP_CodeSniffer)
-* symlink, copy or clone the [Symfony Coding Standard](https://github.com/djoos/Symfony-coding-standard) and symlink, copy or move its `Symfony` subdirectory to `src/Standards/Symfony`
-* symlink, copy or clone this repository to `src/Standards/MO4`
-* from the CodeSniffer repository root run `phpunit --filter MO4`
+See [CONTRIBUTING.md](CONTRIBUTING.md) for information.
```
Comment (mmoll) [inline README.md]: 😃

File: CONTRIBUTING.md (hunk 1 above)
Comment (xalopp) [reply CONTRIBUTING.md]: J'agree

**Comment by mmoll** [comment]
merged, merci @xalopp!

**Pull request merged**